edge-modules/load-gen/Settings.cs

```diff
@@ -29,20 +29,31 @@ public class Settings
                     configuration.GetValue<ulong>("messageSizeInBytes", 1024),
                     configuration.GetValue("transportType", TransportType.Amqp_Tcp_Only),
                     configuration.GetValue("outputName", "output1"),
-                    configuration.GetValue("startDelay", TimeSpan.FromSeconds(2)));
+                    configuration.GetValue("testStartDelay", TimeSpan.FromMinutes(5)),
```

Comment:
Do we really need 5 mins delay? (how about say 2 mins?)

Reply:
it depends on platform; RPi generally takes longer time to download all required modules.